adal/build.gradle

```diff
@@ -42,13 +42,14 @@ android {
     }
 
     lintOptions {
-        abortOnError false
+        abortOnError true
         disable 'MissingPermission'
         disable 'LongLogTag'
         disable 'DefaultLocale'
         disable 'UnusedResources'
         disable 'GradleDependency'
         disable 'GradleOverrides'
+        disable 'OldTargetApi'
```

Comment:
>OldTargetApi [](start = 17, length = 12)

can you remove this from here and add @SupressLint to the place where you are hitting error